### PR TITLE
update flags to be verbose, add unittest.skip to temporarily choose w…

### PR DESCRIPTION
**Pull request by jldeen**
- update flags to be verbose
- add unittest.skip to temporarily chose which test to run (due to flaky managed env setup by container apps) 

Signed-off-by: Jessica Deen <jessicadeen@me.com>

### CODE REVIEW

**Comment by jldeen** [comment]
Addresses issue #1

File: src/containerapp-preview/azext_containerapp_preview/tests/latest/test_containerapp_scenario.py [hunk 1]
```diff
@@ -49,10 +54,10 @@ def test_containerapp_compose_create_with_external_ingress(self, resource_group)
         docker_compose_file.close()
 
         self.kwargs.update({
-            'environment': 'test1'
+            'environment': self.create_random_name(prefix='containerapp-preview', length=24)
```
Comment (smurawski) [inline src/containerapp-preview/azext_containerapp_preview/tests/latest/test_containerapp_scenario.py:57]: I was just reading about this, I wanted to try this and see how it worked.  Thanks!

Comment (jldeen) [reply src/containerapp-preview/azext_containerapp_preview/tests/latest/test_containerapp_scenario.py:57]: :)

File: src/containerapp-preview/azext_containerapp_preview/custom.py [hunk 2]
```diff
@@ -46,7 +46,7 @@ def create_containerapps_from_compose(cmd,
         # Internal Ingress Check
         elif service.expose is not None:
             ingress_type = "internal"
-            target_port = int(service.expose.strip("''[]''"))
+            target_port = service.expose[0]
```
Comment (smurawski) [inline src/containerapp-preview/azext_containerapp_preview/custom.py:49]: This makes the current use case work, but we'll have to add some prompting for choice when multiple expose ports are defined (see #12 )

**Review by smurawski — APPROVED** [review]

**Pull request merged**